ui.frontend/src/GuideBridge.js

```diff
@@ -197,15 +197,49 @@ export default class GuideBridge {
         }
     }
 
+    /**
+     * @summary Hides all the submit buttons present in the Adaptive Form
+     *
+     */
     hideSubmitButtons() {
-        //TODO: implement it later. NO-OP for now.
+        if (this.isConnected()) {
+            this.#hideButtons('submit');
+        } else {
+            throw new Error("GuideBridge is not connected");
+        }
     }
 
-    hideSaveButtons() {
-        //TODO: implement it later. NO-OP for now.
+    /**
+     * @summary Hides all the reset buttons present in the Adaptive Form
+     *
+     */
+    hideResetButtons() {
+        if (this.isConnected()) {
+            this.#hideButtons('reset');
+        } else {
+            throw new Error("GuideBridge is not connected");
+        }
     }
 
-    hideResetButtons() {
+    #hideButtons(buttonType) {
+        let formModel = this.getFormModel();
+        formModel.visit((field) => {
+            if (field.properties["fd:buttonType"] === buttonType) {
+                field.visible = false;
+                field.subscribe((action) => {
```

Comment:
Why do we need subscribe here ?

Reply:
we want to hide the submit button always if this API is called.
even in the case someone has written a conditional rule to show it.

Reply:
we are not checking for the buttonType to be "submit" . do we need to add the subscribe code in case buttonType is submit or for all buttonTypes?

Reply:
only buttontype field is subsribed.